### PR TITLE
Scanpy update 1.9.6

### PR DESCRIPTION
**Pull request by pavanvidem**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pavanvidem** [comment]
This PR is based on https://github.com/galaxyproject/tools-iuc/pull/5404

File: tools/scanpy/cluster_reduce_dimension.xml [hunk 1]
```diff
@@ -612,23 +610,23 @@ Cluster cells using the Leiden algorithm (Traag et al, 2018), an improved versio
...
+<https://icb-scanpy.readthedocs-hosted.com/en/stable/api/scanpy.pp.pca.html>`__
 
 Computes PCA (principal component analysis) coordinates, loadings and variance decomposition, using `tl.pca`
 ============================================================================================================
 
 @CMD_pca_outputs@
 
 More details on the `tl.pca scanpy documentation
-<https://icb-scanpy.readthedocs-hosted.com/en/@version@/api/scanpy.tl.pca.html>`__
+<https://icb-scanpy.readthedocs-hosted.com/en/stable/api/scanpy.tl.pca.html>`__
```
Comment (bgruening) [inline tools/scanpy/cluster_reduce_dimension.xml:629]: I guess version was better? Stable will refere to a version that is not the galaxy version at some point.

Comment (pavanvidem) [reply tools/scanpy/cluster_reduce_dimension.xml:629]: Those version specific links do not exist anymore :(

Comment (bgruening) [reply tools/scanpy/cluster_reduce_dimension.xml:629]: ok.

**Comment by pcm32** [comment]
It would be great if we can add here the gene_symbols field changes that we had for plotting here: https://github.com/galaxyproject/tools-iuc/pull/5206

**Comment by pavanvidem** [comment]
> It would be great if we can add here the gene_symbols field changes that we had for plotting here: #5206

Sure, thanks!

**Comment by pavanvidem** [comment]
Ready. All the tests are passing. Large test data (>1MB & <4MB ) already exists from the previous versions.

**Comment by bgruening** [comment]
Thanks @pavanvidem. Great work!

**Comment by pavanvidem** [comment]
Thanks you merging! and thanks to @ivelet @pcm32  for their contributions

**Pull request merged**